src/ui/widgets/PitServiceWidget/PitServiceWidget.module.scss

```
@@ -2,6 +2,10 @@
   display: flex;
   flex-direction: column;
   gap: sp(xs);
+  // Sole child in the default column layout, and the growing half of the row
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)